### ISSUE TITLE
Add ability to focus on operation expand/collapse button

### ISSUE THREAD

**Issue by glowcloud** [swagger-api/swagger-ui#10708]
It is not possible to focus on the operation expand/collapse button (marked in red on the attached image). It is possible to expand/collapse the operation by focusing and interacting with the overall operation button (marked in blue), but the other button is not focusable. 

<img width="1445" height="79" alt="Image" src="https://github.com/user-attachments/assets/78a69a26-ee14-4880-a1b8-e82fb79d54ff" />

This task covers issues originally reported in:
- https://github.com/swagger-api/swagger-ui/issues/7350
